store/tikv/region_cache.go

```diff
@@ -69,13 +69,19 @@ type Region struct {
 type RegionStore struct {
 	workStoreIdx int32    // point to current work peer in meta.Peers and work store in stores(same idx)
 	stores       []*Store // stores in this region
+	epochs       []uint32 // snapshots of store's fail epochs, need reload when `epochs[curr] != stores[cur].failEpoch`
```

Comment:
Is it `RegionEpoch`?

Reply:
no, it just a "store failed count", - - maybe need a better name.

Reply:
Yes, we need a better name, `epoch` may confuse us.